test/integration/select_test.lua

```diff
@@ -77,6 +78,54 @@ pgroup.test_not_valid_value_type = function(g)
     t.assert_str_contains(err.err, "Supplied key type of part 0 does not match index part type: expected unsigned")
 end
 
+pgroup.test_select_nil = function(g)
+    local space = 'customers'
+    local crit_log = "C> empty or nil `select` call on user space=" .. space
+    local valid_condition = {'>=', 'id', 2}
+    local tests = {
```

Comment:
Such tests will fail at first invalid assert. I prefer taking out parametrized data and creating separate cases for each entry (like this: https://github.com/tarantool/crud/blob/20023a9a6faab1976997534e11c3b71f24d9061f/test/unit/stats_test.lua#L93-L209), so each failed case will fail and each successful case will succeed.

Reply:
Great idea, done.